### ISSUE TITLE
paddle1.8.5gpu  paddleHub1.8.3 训练不报错，也不打印日志

### ISSUE THREAD

**Issue by cap2hb** [PaddlePaddle/Paddle#30748]
GPU的利用率一直在0~1之间徘徊。把训练数据减少到10条，依旧没有反应。程序一直停滞在：
C:\Users\Cap\miniconda3\envs\tf15\lib\site-packages\paddle\fluid\clip.py:779: UserWarning: Caution! 'set_gradient_clip' is not recommended and may be deprecated in future! We recommend a new strategy: set 'grad_clip' when initializing the 'optimizer'. This method can reduce the mistakes, please refer to documention of 'optimizer'.
  warnings.warn("Caution! 'set_gradient_clip' is not recommended "
[2021-01-28 14:53:12,983] [    INFO] - Strategy with warmup, linear decay, slanted triangle learning rate, weight decay regularization, 
C:\Users\Cap\miniconda3\envs\tf15\lib\site-packages\paddle\fluid\executor.py:1093: UserWarning: There are no operators in the program to be executed. If you pass Program manually, please use fluid.program_guard to ensure the current Program is being used.
  warnings.warn(error_info)
[2021-01-28 14:53:13,052] [    INFO] - Try loading checkpoint from lcqmc_pointwise_matching\ckpt.meta
[2021-01-28 14:53:13,052] [    INFO] - PaddleHub model checkpoint not found, start from scratch...
[2021-01-28 14:53:13,151] [    INFO] - PaddleHub finetune start
再也没有下文了，请问是什么原因

**Comment by cap2hb** [comment]
代码是你们官方的实例：
from paddlehub.dataset.base_nlp_dataset import TextMatchingDataset
import paddlehub as hub
import os


class PointwiseDataset(TextMatchingDataset):
    # Pointwise数据集
    # 通过传入不同数据集文件夹，来使用不同数据集
    """
        bq_corpus = PointwiseDataset('data/bq_corpus', tokenizer=tokenizer, max_seq_len=max_seq_len)
        paws-x-zh = PointwiseDataset('data/paws-x-zh', tokenizer=tokenizer, max_seq_len=max_seq_len)
        lcqmc = PointwiseDataset('data/lcqmc', tokenizer=tokenizer, max_seq_len=max_seq_len)
    """

    def __init__(self, dataset_dir, tokenizer=None, max_seq_len=None):
        self.dataset_dir = dataset_dir
        super(PointwiseDataset, self).__init__(
            is_pair_wise=False,  # 文本匹配类型，是否为pairwise
            base_path=self.dataset_dir,
            train_file="train.tsv",  # 相对于base_path的文件路径
            dev_file="dev.tsv",  # 相对于base_path的文件路径
            predict_file="test.tsv",
            train_file_with_header=False,
            dev_file_with_header=False,
            predict_file_with_header=False,
            label_list=["0", "1"],
            tokenizer=tokenizer,
            max_seq_len=max_seq_len)


# =====================================================================================================================
#                                                     模型训练配置开始
# =====================================================================================================================
# Pointwise文本匹配
# 数据集设置，通过更换数据集文件夹更换数据集
dataset_dir = 'data/lcqmc'
dataset_name = dataset_dir.split('/')[-1]

# 加载语义模型，可更换其他语义模型比如bert、robert等
module = hub.Module(name="ernie")

# 加载tokenizer
tokenizer = hub.BertTokenizer(
    vocab_file=module.get_vocab_path(),
    tokenize_chinese_chars=True
)

# 定义数据集
dataset = PointwiseDataset(
    dataset_dir,
    tokenizer=tokenizer,
    max_seq_len=128
)

# 设置优化策略
strategy = hub.AdamWeightDecayStrategy(
    learning_rate=5e-5,
    lr_scheduler="linear_decay",
    warmup_proportion=0.1,
    weight_decay=0.01,
    optimizer_name="adam"
)

# 设置训练参数
config = hub.RunConfig(
    log_interval=20,
    eval_interval=500,
    use_pyreader=True,
    use_data_parallel=True,
    save_ckpt_interval=500,
    use_cuda=True,
    checkpoint_dir='%s_pointwise_matching' % dataset_name,
    num_epoch=10,
    batch_size=4,
    strategy=strategy)

# 模型上下文
inputs, outputs, program = module.context(
    trainable=True,
    max_seq_len=128,
    num_slots=2
)

# 模型输出
query = outputs["sequence_output"]
title = outputs['sequence_output_2']

# 训练任务
pointwise_matching_task = hub.PointwiseTextMatchingTask(
    dataset=dataset,
    query_feature=query,
    title_feature=title,
    tokenizer=tokenizer,
    config=config,
    network='lstm',
    metrics_choices=['acc']
)

# 开始训练
run_states = pointwise_matching_task.finetune_and_eval()


# =====================================================================================================================
#                                                     模型训练配置结束
# =====================================================================================================================


# =====================================================================================================================
#                                                     文本相似度预测开始
# =====================================================================================================================
# 测试集读取器
def get_testset(dataset_dir):
    datas = []
    with open(os.path.join(dataset_dir, 'test.tsv'), 'r', encoding='UTF-8') as f:
        for line in f:
            line = line[:-1].split('\t')
            datas.append(line)
    return datas


# pointwise模型
dataset_dir = 'data/lcqmc'
dataset_name = dataset_dir.split('/')[-1]
testset = get_testset(dataset_dir)

results = pointwise_matching_task.predict(
    data=testset,
    max_seq_len=128,
    label_list=['0', '1'],
    return_result=True,
    accelerate_mode=False)

with open('%s_result.tsv' % dataset_name, 'w', encoding='UTF-8') as f:
    f.write('index\tprediction\n')
    for i, result in enumerate(results):
        f.write('%d\t%s\n' % (i, result))
# =====================================================================================================================
#                                                     文本相似度预测结束
# =====================================================================================================================

**Comment by zhwesky2010** [comment]
@cap2hb 您好，请问paddle安装对了吗？fluid.install_check.run_check() 有问题吗

**Comment by cap2hb** [comment]
> @ cap2hb您好，请问paddle安装对了吗？fluid.install_check.run_check（）有问题吗

安装正常的，没有任何问题，paddle1.8.5gpu paddleHub1.8.3就是跑不起来。Nvidia-smi查看显存有被使用，但是使用率一直在0~~%1之间跳动。我的显卡是rtx2070，cuda10.0 、cudnn7.6 没有版本问题。之后改用paddlepaddle2.0和paddlehub2.0，cuda11.0、cudnn11.0可以正常运行。请问是因为cuda10.0对于rtx2070版本太旧造成的吗？
